api_specifications/case-api-0.1.0.yaml

```
@@ -66,6 +66,34 @@
               schema:
                 $ref: >-
                   https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+  /case/v1/_search:
+    post:
+      tags:
+        - Case
+      summary: Fetch cases
+      description: | 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)